dist/install_elinks.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [[ "$(sha256sum gpm-libs-1.20.7-17.el8.x86_64.rpm | cut -d' ' -f1)" == "9fe5b0c5fee372362d423e17b56f01d1c15cc0725bd989bdfc742f30c1119e0b" ]]
```

Comment:
I'd prefer to because if the files ever get re-downloaded or changed, this is an easily-retrievable record of change in git (probably harder to miss than a whole-file change) *and* a sanity check that the container image is really built the way we want. Both reasons are very weak, given the fact that it's all in the same repo. I can remove the check if you want.